### ISSUE TITLE
Send feedbacks to email instead of storing them to the database

### ISSUE THREAD

**Issue by manishram** [thenewboston-developers/Website-API#129]
Instead of storing feedbacks to the database [[#116](https://github.com/thenewboston-developers/Website-API/pull/116)], that we are receiving from the feedback form on this page: https://thenewboston.com/faq/all, we need to send those feedbacks as email to `contact [at] thenewboston [dot] com`

---

**Email template:**

**Subject:** User feedback from Website

-----------Email Body------------
**Name:** `< Name from the form >`
**Email:** `< Email from the form >`
**Feedback:** `< User Feedback >`
----------Email Body End---------

**Reply-to:** `< User's email from the form >`

---
So basically we need to revert back to this PR: [[#116](https://github.com/thenewboston-developers/Website-API/pull/116)] and then implement the above-mentioned approach.

Let me know if something is unclear.

**Comment by ThaDeveloper** [comment]
@manishram We need a default from email

**Comment by manishram** [comment]
Hey @ThaDeveloper let's keep it the way it is right now. Closing this for now.
